include/flatbuffers/flatbuffers.h

```diff
@@ -1762,6 +1797,7 @@ class FlatBufferBuilder {
     return EndVectorOfStructs<T>(vector_size);
   }
   #endif
+  #endif  // !FLATBUFFERS_NO_STL
```

Comment:
May be lift up `Offset<Vector<const T *>> CreateVectorOfStructs()` at line 1813 then merge several `FLATBUFFERS_NO_STL` blocks into one block?
`CreateVectorOfStructs(const T *v, size_t len)` at line 1756 also can be lifted up.

Reply:
Done. Merged a few others too.